integration-libs/s4-service/order/core/connector/reschedule-service-order.connector.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { inject, Injectable } from '@angular/core';
+import { RescheduleServiceOrderAdapter } from './reschedule-service-order.adapter';
+import { ServiceDetails } from '@spartacus/s4-service/root';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class RescheduleServiceOrderConnector {
+  protected rescheduleServiceOrderadapter = inject(
```

Comment:
Make camel case rescheduleServiceOrderadapter.Also see if this can be made private?

Reply:
Done.